src/Faker/Generator.php

```diff
@@ -184,7 +184,7 @@ public function seed($seed = null)
         if ($seed === null) {
             mt_srand();
         } else {
-            mt_srand($seed);
+            mt_srand(intval($seed));       
```

Comment:
Why not just use `(int) $seed`. It's exactly the same, but has better performance...